appgate/files.py

```diff
@@ -27,18 +27,34 @@ class AppgateFile:
     be an AppgateFile instance that will be able to load its value
     """
 
-    def __init__(self, value: Dict, entity_name: str) -> None:
+    def __init__(self, value: Dict, entity_name: str, field_name: str) -> None:
         self.value = value
         self.entity_name = entity_name
+        self.field_name = field_name
         self.api_version = os.getenv("APPGATE_API_VERSION")
 
     def load_file(self) -> str:
         raise NotImplementedError()
 
 
+def file_path(value: Dict, field_name: str, entity_name: str) -> str:
```

Comment:
This could use a docstring, its the http path for the file in the SDP API right?

Reply:
yes, this is where the path part is computed

Reply:
It already has a docstring